src/Mod/RuleItem.h

```diff
@@ -210,7 +210,7 @@ class RuleItem
 	std::map<std::string, std::string> _zombieUnitByArmorMale, _zombieUnitByArmorFemale, _zombieUnitByType;
 	std::string _zombieUnit, _spawnUnit;
 	int _spawnUnitFaction;
-	bool _LOSRequired, _underwaterOnly, _landOnly, _psiReqiured, _manaRequired;
+	bool _LOSRequired, _friendlyTargetingAllowed, _enemyTargetingAllowed, _underwaterOnly, _landOnly, _psiReqiured, _manaRequired;
```

Comment:
The attribute names should probably contain info that it only apples to psiamps... and not for example medikits (which use different attributes) or firearms (which don't use any attributes).